starter-code/src/clue.js

```diff
@@ -92,3 +169,52 @@ var roomsArray = [];
 
 // Weapons Collection
 var weaponsArray = [];
+
+charactersArray.push(mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard);
+
+roomsArray.push(dinningRoom,conservatoryRoom, kitchenRoom, studyRoom, libraryRoom, billiarrdRoom, loungeRoom, ballRoom, hallRoom,spaRoom, livingRoom, obervatoryRoom, theaterRoom, guestRoom, patioRoom);
+
+weaponsArray.push(rope, knife, candleStick, dumBell, poison, axe, bat, trophy, pistol);
+
+function randomSelector(array){
+   var random =  Math.floor(Math.random() * (array.length))  /* math.random picks a random number from 1 - 0.9999 
```

Comment:
``` var random =  Math.floor(Math.random() * (array.length + 1))```

Reply:
Oops! Bad correction! If you are using the number as an index it's okay :disappointed: sorry